client/service/src/task_manager.rs

```diff
@@ -123,7 +123,7 @@ pub struct TaskManager {
 impl TaskManager {
  	/// If a Prometheus registry is passed, it will be used to report statistics about the
  	/// service tasks.
-	pub(super) fn new(
+	pub fn new(
```

Comment:
And plus revert the removal of these `(super)`. They were here to prevent people from doing what you just tried doing.